packages/addons/service/fd628/package.mk

```diff
@@ -17,8 +17,8 @@
 ################################################################################
 
 PKG_NAME="fd628"
-PKG_VERSION="1.0"
-PKG_REV="100"
+PKG_VERSION="1.0.1"
+PKG_REV="101"
```

Comment:
`PKG_REV` can be kept at 100 because you have already bumped the version, `PKG_REV` is generally only bumped when there is code changes with no change to the `PKG_VERSION` as often happens with 3rd party add ons.